### DISCUSSION TITLE
GuppyFLO Camera workaround

### DISCUSSION THREAD

**Discussion by revunix** [Guilouz/Creality-Helper-Script-Wiki#478] [General]
I saw this, but it's wrong.
**"For now camera not working remotely and it will be fixed in future release of GuppyFLO."**

You can get the camera to work by adding it directly to Moonraker via the install script.

Tools Menu and then "Enable camera settings in Moonraker"

**Comment by Guilouz** [comment]
Camera working only in local not on remote access even if it’s configured with moonraker.

**Reply by revunix** [reply]
No, this is not correct, it also works remote.

Okay, i had WIFI active, you're right. But for local it works w/o adding the camera to GruppyFLO over Fluidd.

**Reply by Guilouz** [reply]
That’s I have said: For now camera not working remotely and it will be fixed in future release of GuppyFLO.
